Mixpanel/Mixpanel.m

```diff
@@ -2018,15 +2006,15 @@ - (NSString *)description
 
 - (NSDictionary *)collectAutomaticPeopleProperties
 {
-    NSMutableDictionary *p = [NSMutableDictionary dictionary];
+    NSMutableDictionary *p = [NSMutableDictionary dictionaryWithDictionary:@{@"$ios_version": [UIDevice currentDevice].systemVersion,
+                                                                             @"$ios_lib_version": VERSION,
+                                                                             }];
+    NSDictionary *infoDictionary = [NSBundle mainBundle].infoDictionary;
+    p[@"$ios_app_version"] = infoDictionary[@"CFBundleVersion"];
+    p[@"$ios_app_release"] = infoDictionary[@"CFBundleShortVersionString"];
```

Comment:
No, this subscripting syntax for dictionaries does not throw on `nil`: it is not like the subscripting syntax for arrays where it can throw for out of bounds. And `setValue:forKey:` should be avoided as much as possible, as it may have unpredictable behaviour if the key starts with the '@' character.